src/hearings/containers/request-hearing/request-hearing.component.html

```diff
@@ -4,7 +4,8 @@
     <main id="content" role="main">
       <a *ngIf="!isConfirmationPage" href="javascript:void(0)" (click)="onBack()" class="govuk-back-link">Back</a>
       <router-outlet></router-outlet>
-      <button *ngIf="!isCheckAnswerPage && !isConfirmationPage" class="govuk-button" data-module="govuk-button" (click)="onContinue()">Continue</button>
+      <button *ngIf="!isCheckAnswerPage && !isConfirmationPage && !isHearingActualPage" class="govuk-button" data-module="govuk-button" (click)="onContinue()">Continue</button>
+      <button *ngIf="isHearingActualPage" class="govuk-button" data-module="govuk-button" (click)="onSubmit()">Submit hearing details</button>
```

Comment:
after this is extracted to the actuals hearing jouney it shouldn't touch anything on request hearing journey.